cmds/report.c

```diff
@@ -640,21 +834,28 @@ char * convert_sort_keys(char *sort_keys)
 	return new_keys;
 }
 
+char * convert_sort_key(char *sort_keys)
```

Comment:
I know this conversion rule could make restriction to sort key.
Please give me your opinions for this.

Reply:
Did you mean the conversion from '-' to '_' ?  I'm totally ok with that.

Reply:
ok. thank you.